axelar-chains-config/src/utils/getChainArray.js

```diff
@@ -0,0 +1,26 @@
+import fs from 'fs';
+
+/**
+ * Get the chain config for a given environment (Currently only supports 'mainnet', 'testnet' and 'stagenet'). The returned value will be an array of chains instead of key-value pairs.
+ * @param {*} env - The environment to get the chain config for (e.g. 'mainnet', 'testnet', 'stagenet')
+ * @returns {Array} - An array of chain configs
+ */
+export function getChainArray(env) {
```

Comment:
is this for compatibility with the old format?

Reply:
yes

Reply:
cool, we should still switch to the new format for our packages since the old format is clunkier to work with

Reply:
A new format introduces some benefit, like we can just use chain name to access the chain object directly instead for finding it in a loop. However, i think the array format might still useful in some cases like displaying all chains in a webpage.

Reply:
For axelar-local-dev and axelar-examples, there's no issue to use the new format.